### PR TITLE
[Jetsurvey] Using alpha03 Snackbar APIs

### PR DESCRIPTION
**Pull request by florina-muntenescu**
Updating Sign In and Sign Up screens to use `Scaffold`

### CODE REVIEW

File: Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInScreen.kt [hunk 1]
```diff
@@ -122,46 +145,44 @@ fun SignInContent(
...
-                        text = errorText,
-                        style = MaterialTheme.typography.body2
-                    )
-                },
-                action = {
-                    TextButton(
-                        onClick = onDismiss,
-                        contentColor = contentColor()
-                    ) {
+    snackbarHostState.currentSnackbarData?.let { data ->
```
Comment (chrisbanes) [inline Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInScreen.kt:155]: Do you need this? The `snackbarHostState` should handle this for you.

File: Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInScreen.kt [hunk 2]
```diff
@@ -56,36 +57,58 @@ sealed class SignInEvent {
...
-                    onClick = { showSnackbar.value = true },
-                    modifier = Modifier.fillMaxWidth()
-                ) {
-                    Text(text = stringResource(id = R.string.forgot_password))
                 }
             }
         }
+    )
+
+    Stack(modifier = Modifier.fillMaxSize()) {
```
Comment (chrisbanes) [inline Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInScreen.kt:107]: Just checking, should this be at the bottom of the screen?

Comment (florina-muntenescu) [reply Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInScreen.kt:107]: Yes. good catch.

File: Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInScreen.kt (hunk 1 above)
Comment (florina-muntenescu) [reply Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInScreen.kt:155]: `currentSnackbarData` is nullable so when I try to set the `message` I'd end up with a nullable string. 
I could do `snackbarHostState.currentSnackbarData?.message?.let { ... }` in the snackbar text lambda, but what's the point of showing a snackbar without a message?

Comment (chrisbanes) [reply Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInScreen.kt:155]: Ahhh I see. Yeah remove this, and then use the `data` pass into the `snackbar` lambda below

**Review by chrisbanes — APPROVED** [review]

**Pull request merged**